sw/airborne/firmwares/rotorcraft/guidance/guidance_v_adapt.c

```diff
@@ -36,12 +36,17 @@
 /** Initial hover throttle as factor of MAX_PPRZ.
  *  Should be a value between #GUIDANCE_V_ADAPT_MIN_HOVER_THROTTLE and
  *  #GUIDANCE_V_ADAPT_MAX_HOVER_THROTTLE.
- *  It is better to start with low thrust and let it rise as the adaptive filter
- *  finds the vehicle needs more thrust.
+ *  If the nominal hover throttle is defined use it otherwise it is better
+ *  to start with low thrust and let it rise as the adaptive filter finds
+ *  the vehicle needs more thrust.
  */
 #ifndef GUIDANCE_V_ADAPT_INITIAL_HOVER_THROTTLE
+#ifdef GUIDANCE_V_NOMINAL_HOVER_THROTTLE
+#define GUIDANCE_V_ADAPT_INITIAL_HOVER_THROTTLE GUIDANCE_V_NOMINAL_HOVER_THROTTLE
+#else
 #define GUIDANCE_V_ADAPT_INITIAL_HOVER_THROTTLE 0.3
```

Comment:
I would advise against raising the default to 50%
- there is no reason why it should be ideal to hover at 50% (it's even nicer if you need less, i.e. have a better thrust/weight ratio)
- it's better to start at a lower hover throttle and let the adaptive throttle increase it than to start with too much and have it shoot up